src/app/container/container.component.ts

```diff
@@ -197,8 +197,15 @@ export class ContainerComponent extends Entry implements AfterViewInit {
     if (tool) {
       this.tool = tool;
       this.initTool();
+      this.requestAccessHREF$ = this.extendedTool$.pipe(map(data => {
+        if (Object.keys(data).length !== 0 ) {
```

Comment:
Since you only need imgProvider, suggest you just look for that, instead of seeing if it has many properties, one of which may or may not be imgProvider.
```
if (data.imgProvider) {
```